disnake/ui/action_row.py

```diff
@@ -205,6 +208,61 @@ def add_select(
             )
         )
 
+    def add_input_text(
```

Comment:
As I said below (in `ui.Modal.py`), I don't think this is completely necessary? Since you can't have `InputText`s outside of modals.

Reply:
This is completely necessary, just in case someone does
```py
row = disnake.ui.ActionRow(...)
row.add_input_text()
.send_modal(..., components=row)
```